### PR TITLE
ARN required not json, missed by myself. Tested.

### PR DESCRIPTION
**Pull request by ep-93**

### CODE REVIEW

File: terraform/environments/bootstrap/delegate-access/iam.tf [hunk 1]
```diff
@@ -27,7 +27,7 @@ module "ssm-cross-account-access" {
     aws = aws.workspace
   }
   account_id                  = local.environment_management.account_ids["core-shared-services-production"]
-  policy_arn                  = data.aws_iam_policy_document.execution-combined-policy.json
+  policy_arn                  = aws_iam_policy.execution-combined-policy.arn
```
Comment (ep-93) [inline terraform/environments/bootstrap/delegate-access/iam.tf:30]: @davidkelliott I missed the arn here and supplied json... even though it said arn... not sure what I was thinking!

File: terraform/environments/bootstrap/delegate-access/iam.tf [hunk 2]
```diff
@@ -128,10 +128,17 @@ data "aws_iam_policy_document" "trust-relationship-policy" {
   }
 }
 
-data "aws_iam_policy_document" "execution-combined-policy" {
+data "aws_iam_policy_document" "execution-combined-policy-doc" {
   source_policy_documents = concat([data.aws_iam_policy_document.SSM-Automation-Policy.json, data.aws_iam_policy_document.execution-policy.json])
 }
 
+resource "aws_iam_policy" "execution-combined-policy" {
```
Comment (davidkelliott) [inline terraform/environments/bootstrap/delegate-access/iam.tf:135]: Are the cloudformation permissions needed? (in the json above, sorry didn't review this one before :) )

File: terraform/environments/core-shared-services/iam.tf [hunk 3]
```diff
@@ -228,11 +228,19 @@ data "aws_iam_policy_document" "SSM-Automation-Admin-Policy-Trust-Relationship"
   }
 }
 
+resource "aws_iam_policy" "ssm-cross-account-admin-policy" {
+  name        = "SSM-Automation-Execution-Combined-Policy"
```
Comment (davidkelliott) [inline terraform/environments/core-shared-services/iam.tf:232]: Is this the correct name?  Looks like a copy and paste from above

File: terraform/environments/core-shared-services/iam.tf [hunk 4]
```diff
@@ -228,11 +228,19 @@ data "aws_iam_policy_document" "SSM-Automation-Admin-Policy-Trust-Relationship"
...
+  path        = "/"
+  description = "Combined policy for SSM Automation Execution"
```
Comment (davidkelliott) [inline terraform/environments/core-shared-services/iam.tf:234]: As above copy and paste

**Review by davidkelliott — CHANGES_REQUESTED** [review]
Few minor things but other than that I think it looks good 👍

**Review by davidkelliott — APPROVED** [review]
Thanks 👍

**Pull request merged**